### PR TITLE
e2e:chore - test for the information severity flag

### PR DESCRIPTION
**Pull request by lucasbrunozup**
**- What I did**
E2e test for the `--information-severity`

**- How to verify it**
run `make test-e2e` and expected all tests passed.

**- Description for the changelog**
N/A

Signed-off-by: lucas.bruno <lucas.bruno@zup.com.br>

### CODE REVIEW

**Comment by lucasbrunozup** [comment]
When trying to write a more assertive test, using an `INFO` type vulnerability, I identified that the CLI is not showing this severity and reported what happened in issue #785

**Review by iancardosozup — APPROVED** [review]

File: e2e/commands/start/start_test.go [hunk 1]
```diff
@@ -296,4 +296,18 @@ var _ = Describe("running binary Horusec with start parameter", func() {
...
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath:         testutil.JavaScriptExample1,
+				testutil.StartFlagInformationSeverity: "true",
+			}
+		})
+
+		It("Checks if the information severity property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"enable_information_severity\": true`))
+			Expect(session.Out.Contents()).NotTo(ContainSubstring("Horusec not show info vulnerabilities in this analysis"))
```
Comment (nathanmartinszup) [inline e2e/commands/start/start_test.go:310]: Is this correct? This message should only appear when the flag has not been set.

`Horusec not show info vulnerabilities in this analysis`

Comment (lucasbrunozup) [reply e2e/commands/start/start_test.go:310]: Yes, that's why I used `NotTo`

Comment (nathanmartinszup) [reply e2e/commands/start/start_test.go:327]: I get it!

**Review by nathanmartinszup — APPROVED** [review]

**Comment by nathanmartinszup** [comment]
Resolve conflicts. @lucasbrunozup

**Pull request merged**